buddypress/members/single/profile.php

```diff
@@ -268,6 +268,54 @@
         </div>
         <?php endif; ?>
         <?php endif; ?>
+        <?php if($info['campaigns_participated']->display): ?>
+        <?php 
+            $campaigns = get_user_meta($user->ID, 'campaigns', true);
+            $campaign_count = 0;
+        ?>
+        <?php if(is_array($campaigns) && sizeof($campaigns) > 0): ?>
+        <h2 class="profile__heading"><?php print __("Campaigns Participated In", "community-portal"); ?></h2>
+        <div class="profile__card">
+        <?php foreach($campaigns AS $campaign_id): ?>
+        <?php 
+            $campaign = get_post($campaign_id);
+            $description = get_field('card_description', $campaign->ID);
```

Comment:
This line and the next three are throwing the following error: "Trying to get property 'ID' of non-object"

Reply:
Hmmm, what I'm assuming here is you deleted a campaign but the IDs are still being stored.  I added a check to make sure there is an object still.